### ISSUE TITLE
Are you using KubeSphere?

### ISSUE THREAD

**Issue by FeynmanZhou** [kubesphere/kubesphere#4123]
## 🙋‍♂️ Are you using KubeSphere?
If you are using KubeSphere, welcome to share your use case with our community and win some KubeSphere swags (T-shirt, mug, badge, etc.)!

## ✨ Why?
KubeSphere has been widely adopted by many users in a wide range of industries, but it is hard to track usage and scenarios in OSS. If you leave a comment on this issue we can better understand who our users are and how you are using KubeSphere, we would like to deliver a better user experience for YOU! ❤️

- We are running SIG meetings, community meetings, and local meetups. We'd like to invite you to be a part of our community and collaborate more closely with the maintainers/contributors (if you want to)
- We want to help promote your organization and your case study with other users
- We want to show off all the awesome stuff our users are doing

## 👨‍💻‍ How you can help
Submit a comment in this issue to include the following information, which would then also get included in the [community website](https://kubesphere.io/case/):

✍️ Example:

```
- Organization/Company:  KubeSphere
- Website: https://kubesphere.io/
- Country/City: China/Beijing
- How to connect (your email or Twitter at least one please): feynman@kubesphere.io / https://twitter.com/KubeSphere
- Usage scenario: e.g. We use KubeSphere for internal PaaS and DevOps system.
- Status: Development?/Production?
```

![image](https://user-images.githubusercontent.com/40452856/129017954-1c210a72-b54e-4fb4-be87-ebc5e000c232.png)

**Comment by yasir2000** [comment]
Yes am using Kubesphere for 9 months now, its main merit and advantage a total solution for integrated K8s cluster management, it saved 10's of hours configuring mesh, monitoring, tracing, messaging, serverless, CI/CD,...etc all in one box. Kubesphere needs more tooling, bug fixing and performance/resources optimization

**Comment by juan-vg** [comment]
Hello there!

At Geko Cloud we use KubeSphere on many cloud providers (EKS, GKE, ...) in order to provide a friendly management-layer to our customers. It was around one year ago when we begun our research about something more than a simple dashboard, and we found KubeSphere very interesting because it was integrating all together many day-to-day procedures (which we were already doing manually) but in a clicky, easy way. This eases our non-IT customers' handling of the platform, while they're kind of autonomous while operating a K8S cluster, which leads us to cut costs, reduce operative assistance times and therefore, allows us to dedicate quality time to higher level tasks.

Additionally (but before starting to use KubeSphere in a productive way), we managed to have a meet with the KubeSphere team. They are so friendly and committed, and we're having fluent and frequent communication between us. They told us about their future plans (which I can told you are exciting and well-thinked) and we realized we both were aligned, so finally decided to become partners by starting the console localization to our language (Spanish) and contributing as much as we can. We're proud to become part of this family, and now we encourage you to do the same. Let's do it even bigger!

**Comment by damon008** [comment]
- Organization/Company:  leinao
- Website: http://www.leinao.ai/
- Country/City: China/Hefei
- How to connect (your email or Twitter at least one please): shoufaw@leinao.ai / MrNull008
- Usage scenario: e.g. We use KubeSphere for internal PaaS and DevOps system
- Status: Development?/Production? Dev and Test

**Comment by webup** [comment]
- Organization/Company: UISEE
- Website: uisee.com
- Country/City: China/Shanghai
- How to connect (your email or Twitter at least one please): haili.zhang@uisee.com
- Usage scenario: We use KubeSphere mainly in multi-cloud multi-cluster management, especially taking advantage of its monitoring and alerting system to watch out our workloads.
- Status: Already applied KubeSphere in production since v2.1, with v3.1.1 already deployed.

**Comment by dhyanio** [comment]
- Organization/Company: dhyanio
Website: dhyanio.com
Country/City: Delhi/India
How to connect (your email or Twitter at least one please): dhyani.devops@gmail.com
Usage scenario: I am testing Kubesphere, for my multiple kubernetes cluster management. But I have faced some performance issue as compare to other open source option. But I prefer Kubesphere bos, it is using golang and some other libraries. 
Status: Playing for pre-prod. but Also creating our own solution.

**Comment by damon008** [comment]
yes~


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "kubesphere/kubesphere"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月20日(星期五) 下午3:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [kubesphere/kubesphere] Are you using KubeSphere? (#4123)





  
Organization/Company: dhyanio
 Website: dhyanio.com
 Country/City: Delhi/India
 How to connect (your email or Twitter at least one please): ***@***.***
 Usage scenario: I am testing Kubesphere, for my multiple kubernetes cluster management. But I have faced some performance issue as compare to other open source option. But I prefer Kubesphere bos, it is using golang and some other libraries.
 Status: Playing for pre-prod. but Also creating our own solution.
   
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ylmcr7** [comment]
![image](https://user-images.githubusercontent.com/63889450/130411865-2d511343-7fa4-4ff6-8e9e-489c88f96f35.png)

我在我自己的测试环境安装了kubesphere,我感觉页面非常棒，比我在扫码登陆阿里云好的太多了，经过我的测试，我认为，这是一个新的时代，我们使用了kubesphere之后，RD可以自己进行项目管理，分等级的进行操作，还可以把项目打包成一个helm模板，大大降低了运维的维护成本，和上线成本，不过里面还有很多的bug，我还没有找到答案，比如给master节点打label,但是我的测试环境只开了两台node节点，该怎么操作呢？

**Comment by FeynmanZhou** [comment]
> ![image](https://user-images.githubusercontent.com/63889450/130411865-2d511343-7fa4-4ff6-8e9e-489c88f96f35.png)
> 
> 公司或组织：公司上kubesphere调研者
> 官网：
> 城市：北京
> 联系方式 (邮件地址或者微信号中的一个)：[ylmcr7@163.com](mailto:ylmcr7@163.com)
> 应用场景：devops，多集群管理
> 应用状态：测试环境
> 感受：
> 我在我自己的测试环境安装了kubesphere,我感觉页面非常棒，比我在扫码登陆阿里云好的太多了，经过我的测试，我认为，这是一个新的时代，我们使用了kubesphere之后，RD可以自己进行项目管理，分等级的进行操作，还可以把项目打包成一个helm模板，大大降低了运维的维护成本，和上线成本，不过里面还有很多的bug，我还没有找到答案，比如给master节点打label,但是我的测试环境只开了两台node节点，该怎么操作呢？

@ylmcr7 Thanks! Could you leave your comments at [Chinese Forum](https://kubesphere.com.cn/forum/d/5293-kubesphere)?

**Comment by shfpflyawei** [comment]
Organization/Company: jingqi
Website: http://www.jqsoft.net
Country/City: China/Hefei
How to connect (your email or Twitter at least one please): 15201344146@163.com
Usage scenario: At present, I am mainly responsible for promoting containerization within the company. At present, I mainly use kubesphere in the test environment to help me manage the container platform and deploy the containerized Devops process. I feel that multi cluster management and cluster monitoring are very good
Status: Already applied KubeSphere  deployed  in v3.1.1

**Comment by tscswcn** [comment]
Organization/Company: ericsson
Website: http://www.ericsson.com
Country/City: China/Beijing
How to connect (your email or Twitter at least one please): 755200@qq.com
Usage scenario: At present, I am trying kubesphere 3.1
Status: Try KubeSphere deployed in v3.1.1

**Comment by halil-bugol** [comment]
@calvinyv Can you close this issue? I'm opening a new one
